src/metadata.rs

```diff
@@ -235,7 +235,7 @@ pub trait Metadata: MajorObject {
     }
 }
 
-/// Standalone metadata entry, as returned by iterator from [`Metadata::metadata_iter`].
+/// Standalone metadata entry, as returned by iterator from [`Metadata::metadata`].
```

Comment:
Stray bug.

Reply:
Why can't rustdoc warn about this, I wonder?

Reply:
It does, if you remember to run it! 😄.

I guess the more specific question is: should we add this as a CI check?

Reply:
I'd say yes. Are we clean on those warnings now?

Reply:
Yes, should be. Note, I use [this](https://github.com/georust/gdal/blob/master/.cargo/config.toml#L3) (`cargo dto`) when writing docs to help me catch errors in the examples. `rustdoc` isn't enought. Seems we need a combination of that, disallowing `rustdoc` warnings (similar to [this](https://github.com/georust/gdal/blob/master/.cargo/config.toml#L5)).

Reply:
This isn't an error in an example, it's a reference to an undefined symbol. But I'll take your word for it :-).

Reply:
Sorry... was conflating documentation error contexts 🤦 . Both are potential issues.